R/irods-demo.R

```diff
@@ -163,9 +161,9 @@ remove_docker_images <- function() {
 
 start_irods <- function(verbose, recreate = TRUE) {
   if (isTRUE(recreate)) {
-    cmd <- " ; docker compose up -d --force-recreate nginx-reverse-proxy irods-client-http-api irods-client-icommands"
+    cmd <- " ; docker compose up -d --force-recreate "
   } else {
-    cmd <- " ; docker compose up -d nginx-reverse-proxy irods-client-http-api irods-client-icommands"
+    cmd <- " ; docker compose up -d "
```

Comment:
Is this intentional? All of the demo services will be stood up, which might be... a lot...

Reply:
We distribute the package to CRAN with a trimmed down version, as including the full irods demo caused some issues causing rejection by CRAN. The trimmed down version consists of irods catalog provider, irods catalog consumer, http api, icommand and nginx. Hence only docker compose up is required here.

Reply:
Sounds good, thanks. Resolving this.